apps/web/components/chat/message-input.tsx

```
@@ -498,61 +489,62 @@ export function MessageInput({ channelName, draft, replyTo, onCancelReply, onSen
+          </button>
+          <button
+            ref={emojiButtonRef}
+            onClick={() => {
+              setPickerTab("emoji")
+              setShowEmojiPicker((prev) => !prev || pickerTab !== "emoji")
+            }}
+            className="motion-interactive motion-press hover:text-white"
+            style={{ color: pickerTab === "emoji" && showEmojiPicker ? "var(--theme-accent)" : "var(--theme-text-secondary)" }}
+            title="Emoji"
+            aria-label="Insert emoji"
+          >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)